src/xcode/ENA/ENA/Source/Services/PPAC/__tests__/PrivacyPreservingAccessControlServiceMock.swift

```diff
@@ -0,0 +1,24 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+@testable import ENA
+
+final class PrivacyPreservingAccessControlServiceMock: PrivacyPreservingAccessControlService {
+
+	// MARK: - Protocol PrivacyPreservingAccessControl
+
+	override func getPPACToken(_ completion: @escaping (Result<PPACToken, PPACError>) -> Void) {
```

Comment:
I think it would be nice to Fake the DCDevice somehow and injecting it (real or fake) into the service.
By injecting the DCDevice you could replace this and you could also test the DCDevice.current.isSupported handling of the service.

Reply:
I made a small refactoring and added the protocol `DeviceCheckable` this will encapsulate DeviceCheck API.
Also added a `PPADeviceCheckMock` - this makes tests better.
The PPAService Mock is no longer needed, so I could mal apiToken privet again.